src/multifusion.jl

```diff
@@ -0,0 +1,207 @@
+# Rep Z2 ⊕ Rep Z2 ≅ Ising is worked out here
+# 𝒞 = 𝒟 = RepZ2 ≅ {1, ψ}, while ℳ = Vec ≅ {σ}
+# this is mainly meant for testing within TensorKit without relying on MultiTensorKit
+#------------------------------------------------------------------------------#
+
+@enum CatType 𝒞 = 1 ℳ = 3 ℳᵒᵖ = 2 𝒟 = 4
+# possible TODO: get rid of CatType and use Int instead -> prevents the need to export 𝒞 etc
+function Base.getindex(a::CatType, label::Int)
+    return IsingBimod(a, label)
+end
+
+struct IsingBimod <: Sector
+    type::CatType
+    label::Int
+    function IsingBimod(type::CatType, label::Int)
+        if type == 𝒞
+            0 ≤ label ≤ 1 ||
+                throw(ArgumentError("Invalid 𝒞 label for Ising bimodule: $(label)"))
+        elseif type == ℳ
+            label == 0 ||
+                throw(ArgumentError("Invalid ℳ label for Ising bimodule: $(label)"))
+        elseif type == ℳᵒᵖ
+            label == 0 ||
+                throw(ArgumentError("Invalid ℳᵒᵖ label for Ising bimodule: $(label)"))
+        elseif type == 𝒟
+            0 ≤ label ≤ 1 ||
+                throw(ArgumentError("Invalid 𝒟 label for Ising bimodule: $(label)"))
+        end
+        return new(type, label)
+    end
+end
+
+isC(x::IsingBimod) = x.type == 𝒞
+isM(x::IsingBimod) = x.type == ℳ
+isMop(x::IsingBimod) = x.type == ℳᵒᵖ
+isD(x::IsingBimod) = x.type == 𝒟
+
+function isModule(a::IsingBimod)
```

Comment:
CamelCase is not very standard/common for Julia function names. Given the importance of the name `module` in the Julia language, I would opt for something like `ismodcat` or `ismodulecategory`? Longer names are can help with clarity.